files/galaxy/tpv/tools.yml

```diff
@@ -1250,3 +1250,5 @@ tools:
     scheduling:
       require:
         - singularity
+
+  toolshed.g2.bx.psu.edu/repos/bgruening/omark/omark/.*:
```

Comment:
@rlibouba here is missing something, isn't it?

The memory or cores?

Reply:
Should we add it directly to https://github.com/galaxyproject/tpv-shared-database ?

Reply:
@mira-miracoli lets do that every month. We reserve a day where we go over out entries and move them to upstream.